content/700-optimize/400-recommendations/200-queries-on-unindexed-columns.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Queries on unindexed columns'
+metaTitle: 'Optimize Recommendations: Queries on unindexed columns'
+metaDescription: "Learn about the recommendation provided by Optimize for queries on unindexed columns."
+tocDepth: 3
+toc: true
+---
+
+The following queries targeting the `user` model have a [`where` property](/orm/prisma-client/queries/filtering-and-sorting) for filtering on columns that do not have indexes.
+
+```ts
+await prisma.user.findFirst({
+   where: {
+      name: "Marc"
+   }
+})
+
+await prisma.user.findFirst({
+   where: {
+      name: "Jon"
+   }
+})
+
+await prisma.user.count({
+   where: {
+      name: "Nikolas"
+   }
+})
+```
+
+## Why is this a problem?
+
+An index allows the database to retrieve data more quickly. An index is similar to an index in a book: it helps you quickly locate the information you're looking for without having to go through every page.
```

Comment:
An index in a database allows for quicker data retrieval, similar to how an index in a book helps you locate information without having to go through every page.